lasertagger/run_lasertagger.py

```diff
@@ -127,6 +127,11 @@
 flags.DEFINE_string("embedding_type", None, "Type of embedding. ['Normal', 'POS', " 
                     "'POS_concise', 'Sentence']")
 
+
+flags.DEFINE_float("add_tag_loss_weight", 1, "The weight of loss of add tags.")
```

Comment:
s/The weight of loss of add tags/The weight of loss for adding tags

ditto

Reply:
Done